server/server.js

```diff
@@ -694,6 +694,12 @@ let needSetup = false;
                 bean.authMethod = monitor.authMethod;
                 bean.authWorkstation = monitor.authWorkstation;
                 bean.authDomain = monitor.authDomain;
+                bean.grpUrl = monitor.grpUrl;
```

Comment:
```suggestion
                bean.grpcUrl = monitor.grpcUrl;
```

Reply:
I have fixed it, thank you for your suggestion <3